packages/headless/src/controllers/commerce/product-listing/facets/headless-product-listing-facet-options.ts

```diff
@@ -6,7 +6,7 @@ import {
 } from '../../../../state/state-sections';
 import {CoreCommerceFacetOptions} from '../../core/facets/headless-core-commerce-facet';
 
-const facetResponseSelector = (
+export const facetResponseSelector = (
```

Comment:
A tiny bit out of scope, but we could turn this into a real Redux selector and reap the sweet benefits of memoization!

Reply:
Will do this in a follow-up PR!